src/AdoPat/ITokensHttpClientWrapper.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Authentication.AdoPat
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.VisualStudio.Services.DelegatedAuthorization;
+
+    /// <summary>
+    /// An interface for a transparent wrapper of <see cref="TokensHttpClient"/>.
+    /// </summary>
+    public interface ITokensHttpClientWrapper
+    {
+        /// <summary>
+        /// [Preview API] Creates a new personal access token (PAT) for the requesting user.
+        /// </summary>
+        /// <param name="patTokenCreateRequest">(Undocumented).</param>

```

Comment:
How are we going to handle the "undocumented" stuff. Make our best assumption or reach out to someone who knows about these APIs?

Reply:
We can make a best assumption here, I think. I put `(Undocumented).` because StyleCop wanted *something*, but the source code for the library just has
```csharp
/// <param name="patTokenCreateRequest"></param>
```
but I also think it's largely self-explanatory what these undocumented things are.

Reply:
@shalinikhare27: I took a stab at documenting these items while still being honest about the fact that the underlying library says nothing about them.